dash/development/_py_components_generation.py

```diff
@@ -55,7 +57,10 @@ def generate_class_string(
     _namespace = '{namespace}'
     _type = '{typename}'
     @_explicitize_args
-    def __init__(self, {default_argtext}):
+    def __init__(
+        self,
+        {default_argtext}
+    ):
```

Comment:
I wonder, now that we're Py3-only, maybe we should call `black` on the generated files as part of the generation process? We already do that in the core component packages as a separate step.

Reply:
added issue #2965 to record this